react/components/Button/index.js

```diff
@@ -1,23 +1,29 @@
-import React, { Component } from 'react'
+import React, { Component, Fragment } from 'react'
 import PropTypes from 'prop-types'
 
 import Spinner from '../Spinner'
 
 class Button extends Component {
+  state = {
+    shouldBeDisabled: false,
```

Comment:
Disagree, this adds up way too much logic for a component that should be _dumb_.